docs/project-development-environment.md

```diff
@@ -2,7 +2,19 @@
 
 ## Local prerequisites
 
-Docker and docker-compose is required to be installed.
+| dependency | supported versions | Installation recommendation for Ubuntu 14.04 |
+| ---------- | ------------------ | ---------------------------------------------|
+| <a name='dependency_docker'>docker</a> | 1.10 (1.9 may still work, too) | Follow [the official Docker installation instructions for Linux](https://docs.docker.com/linux/step_one/) |
```

Comment:
I decided to line-wrap commands (2b6566926c52f1ce5642b621ed0de773a09504f8) instead of denormalizing the table. One that does not suffice anymore, I'd extract the installation recommendations to sections outside the table and link to those from the table.